marine/marine.py

```diff
@@ -81,9 +86,13 @@ def filter_and_parse(
                 bpf, display_filter, encoded_fields, encapsulation_type
             )
             if filter_id < 0:
-                raise ValueError(
-                    err
-                )  # TODO: create custom exception for every error type
+                if filter_id == -1:
```

Comment:
I think we should continue the trend of importing consts from the `.so` file, and also import these consts.